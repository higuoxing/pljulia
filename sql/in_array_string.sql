CREATE FUNCTION julia_in_array_text(x text[])
RETURNS text AS $$
    x[1]
$$ LANGUAGE pljulia;

CREATE FUNCTION julia_in_array_varchar(x varchar[])
RETURNS varchar AS $$
    x[1]
$$ LANGUAGE pljulia;

SELECT julia_in_array_text(ARRAY['Apple', 'Banana', 'Orange']);

SELECT julia_in_array_varchar(ARRAY['Apple', 'Banana', 'Orange']);

DROP FUNCTION julia_in_array_text(text[]);

DROP FUNCTION julia_in_array_varchar(varchar[]);
