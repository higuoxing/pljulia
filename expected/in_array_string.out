CREATE FUNCTION julia_in_array_text(x text[])
RETURNS text AS $$
    x[1]
$$ LANGUAGE pljulia;
CREATE FUNCTION julia_in_array_varchar(x varchar[])
RETURNS varchar AS $$
    x[1]
$$ LANGUAGE pljulia;
SELECT julia_in_array_text(ARRAY['Apple', 'Banana', 'Orange']);
 julia_in_array_text 
---------------------
 Apple
(1 row)

SELECT julia_in_array_varchar(ARRAY['Apple', 'Banana', 'Orange']);
 julia_in_array_varchar 
------------------------
 Apple
(1 row)

DROP FUNCTION julia_in_array_text(text[]);
DROP FUNCTION julia_in_array_varchar(varchar[]);
